script/build/update-gomod.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
I'm not a huge fan of hiding this in scripts like this. This is simple enough to leave in the makefile.

Reply:
all rightly! switching.